Sources/PrimerSDK/Classes/Data Models/PaymentMethodViewModel.swift

```diff
@@ -0,0 +1,135 @@
+//
+//  PaymentMethodViewModel.swift
+//  PrimerSDK
+//
+//  Created by Evangelos Pittas on 7/10/21.
+//
+
+import Foundation
+
+class PaymentMethodConfigViewModel {
+    
+    var config: PaymentMethodConfig
+    
+    lazy var title: String = {
```

Comment:
we should exploring splitting out these type of iterations to make the view models less bloated. Wdyt?